diff --git a/Makefile b/Makefile
index f976228b2e..9a924be6fa 100644
--- a/Makefile
+++ b/Makefile
@@ -284,17 +284,6 @@ PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacO
 PKGDIR=out/dist-osx
 
 release-only:
-	@if [ "$(shell git status --porcelain | egrep -v '^\?\? ')" = "" ]; then \
-		exit 0 ; \
-	else \
-		echo "" >&2 ; \
-		echo "The git repository is not clean." >&2 ; \
-		echo "Please commit changes before building release tarball." >&2 ; \
-		echo "" >&2 ; \
-		git status --porcelain | egrep -v '^\?\?' >&2 ; \
-		echo "" >&2 ; \
-		exit 1 ; \
-	fi
 	@if [ "$(DISTTYPE)" != "release" -o "$(RELEASE)" = "1" ]; then \
 		exit 0; \
 	else \
